docs/nlu/choosing-a-pipeline.rst

```diff
@@ -68,6 +68,25 @@ We do not recommend that you use these; mitie support is likely to be deprecated
     You need to provide enough data for both intents and entities.
 
 
+Class imbalance
+---------------
+
+In order to mitigate class imbalance problem, we introduced ``balanced`` batching strategy for
+``supervised_embeddings`` pipeline. This algorithm ensures that all classes are represented
```

Comment:
```suggestion
This algorithm ensures that all classes are represented
```

Reply:
but it is only for embed intent classifier, it doesn't work for space one